pkg/pss/pss.go

```diff
@@ -128,10 +128,11 @@ func (p *pss) topics() []Topic {
 }
 
 // TryUnwrap allows unwrapping a chunk as a trojan message and calling its handlers based on the topic.
-func (p *pss) TryUnwrap(ctx context.Context, c swarm.Chunk) {
+func (p *pss) TryUnwrap(c swarm.Chunk) {
 	if len(c.Data()) < swarm.ChunkWithSpanSize {
 		return // chunk not full
 	}
+	ctx := context.Background()
 	topic, msg, err := Unwrap(ctx, p.key, c, p.topics())
```

Comment:
i dont understand why we need a context here. consider removing it from `Unwrap` altogether.

Reply:
As context is removed from TryUnwrap, it should be removed from Unwrap also, as it serves no purpose now. No cancelation is possible. I would argue that cancelation has its value here, but not so significant. I think that context can be removed from Unwrap.